data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+					goto celebrationrehearsed
+				`	"You said 'virtually.' Who is omitted?"`
+			`	Taely scowls a bit at this. "Only those guilty of the most serious crimes cannot stand as a lead." She pauses, and continues with an amused tone. "Even children take the lead now and then, usually with a parent or sibling to guide them." She smiles at that, then gestures for you to continue asking your questions.`
+			choice
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+				`	"What constitutes your most serious crimes?"`
+			`	Taely gestures harshly. "Crimes against society, of course. Acting against the best interest of our collective wellbeing is the single most destructive and senseless act someone can do."`
+			choice
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label askcrying
+			`	Taely gestures gently. "Those were friends of the crew of a ship that was destroyed while studying a black hole. They acquired ground-breaking data from it, but perished in the process. Their friends and families are proud of them, of course, but that does not change the fact that they are lost to us mortals."`
+			choice
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+				`	"Who leads these ceremonies?"`
+					goto ceremonyleader
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label awestruck
+			`	Taely nods in agreement and simply gestures expansively at the cavern around you, giving you a few moments to contemplate it. Eventually she gestures asking if you have any further questions.`
+			choice
+				`	"I think that is all for now."`
+					goto celebrationrequest
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"Who leads these ceremonies?"`
+					goto ceremonyleader
+				`	"That seemed both well-rehearsed and very disorganized."`
+					goto celebrationrehearsed
+			label celebrationrequest
+			`	Taely strides over to a console near the door and retrieves a crystal from a compartment within the console. She slips it into a case and turns to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present can experience it too. This recording is destined for those working in Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`
```

Comment:
```suggestion
			`	Taely strides over to a console near the closest door and retrieves a crystal from a compartment within. She slips it into a case and turns back to you. "Our celebrations are something that bind us together, and we share in both the memory of the departed and the hope for the future. As such, we keep recordings of them so that those who are not present at a particular event may still experience it too. This recording is destined for those working in Postverta, as a friend of one of those being mourned today is stationed there. Could you deliver this to them please?"`
```